lib/irb/workspace.rb

```diff
@@ -108,8 +108,10 @@ def initialize(*main)
     # <code>IRB.conf[:__MAIN__]</code>
     attr_reader :main
 
-    def load_commands_to_main
-      main.extend ExtendCommandBundle
+    def load_helper_methods_to_main
+      if ExtendCommandBundle.has_helper_method? && !(class<<main;ancestors;end).include?(ExtendCommandBundle)
```

Comment:
I think it's fine to extend the context even if `ExtendCommandBundle` has no methods as it'll simplify tests (e.g. we don't need 2 cases to test workspace commands). We'll also come up with a better solution for this when we support helper methods.

Reply:
Done!
I think it's beneficial to keep main object clean as possible, but I'll consider it later after helper method support